### PR TITLE
Added github repo links and sample project on main page

### PR DESCRIPTION
**Pull request by 0bserver07**
In reply to issue #5 

There seems to be only 1 project so far with a public repository, or I couldn't find them with the Project title given on Projects page.

Later to be added.

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 def parse_project_file(fpath):
     f = open(fpath)
     metadata = {}
-    headers = {'title', 'date', 'category', 'contact', 'tagline', 'mailing_list'}
+    headers = {'title', 'date', 'category', 'contact', 'tagline', 'mailing_list','repository'}
```
Comment (fchollet) [inline build.py]: Space after comma

**Review by fchollet — CHANGES_REQUESTED** [review]
Please do not check anything into `website/`, that content is autogenerated. Added content should go in `templates/`, `'build.py`, etc.

**Comment by 0bserver07** [comment]
1. fixed the /website (my bad first time using this py static site, usually Jekyll).
2. added the ai-on-example repo link.
3. added social-bot- repo link (people to find it)
4. added Gitter to main page and readme (I wasn't quite sure about the way I linked them though)

**Review by fchollet — CHANGES_REQUESTED** [review]
Mention of the example project and the Gitter channel should go in /process. The landing page is for high-level info. Thanks!

**Comment by 0bserver07** [comment]
All good now, wasn't sure if I should remove the gitter from readme.md or not though. (I thought it might help people when they find it on Github)

**Comment by fchollet** [comment]
LGTM, thanks!

**Pull request merged**